sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/CodecStreams.scala

```diff
@@ -86,4 +88,11 @@ object CodecStreams {
       .map(_.getDefaultExtension)
       .getOrElse("")
   }
+
+  /** Creates an input stream from the path. */
```

Comment:
this doesn't tell the difference between this method and `createInputStream(config: Configuration, file: Path)`.

We also need a better name for this method.